bdb/rep.c

```diff
@@ -2397,16 +2440,29 @@ static void got_new_seqnum_from_node(bdb_state_type *bdb_state,
             static unsigned long long count = 0;
             count++;
 
-            if (bdb_state->attr->wait_for_seqnum_trace && (now = time(NULL)) > lastpr) {
-                logmsg(LOGMSG_USER, "%s: rejecting seqnum from %s because gen is "
-                                "%u, i want %u, count=%llu\n",
-                        __func__, host, seqnum->generation, mygen, count);
+            if (seqnum_trace && (now = time(NULL)) > lastpr) {
+                logmsg(LOGMSG_USER,
+                       "%s: rejecting seqnum from %s because gen is "
+                       "%u (low), i want %u, count=%llu\n",
+                       __func__, host, seqnum->generation, mygen, count);
                 lastpr = now;
             }
             return;
         }
 
         if (seqnum->generation > mygen) {
+            static unsigned long long count = 0;
```

Comment:
To avoid needless statics, can this new section move above, at which point the diff would be to change the above if:
if (seqnum->generation != mygen) {...(fix low high)...}

Reply:
Nevermind, we return in the section above.

Reply:
grr, i think the two can be merged : ) but use your judgement, sorry for noise.